webapp/package-lock.json

```diff
@@ -2625,7 +2625,7 @@
     "amdefine": {
       "version": "1.0.1",
       "resolved": "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz",
-      "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
+      "integrity": "sha512-S2Hw0TtNkMJhIabBwIojKL9YHO5T0n5eNqWJ7Lrlel/zDbftQpxpapi8tZs3X1HWa+u+QeydGmzzNU0m09+Rcg==",
```

Comment:
Accidental changes? Maybe part of cherrypick though

Reply:
I ran full `make deploy` after the merge to ensure everything works together, and npm never misses the opportunity to make some changes. 

about integrity hashes, should we ignore that?

Reply:
> about integrity hashes, should we ignore that?

I'm honestly not sure. @hmhealey Do you have any thoughts on this? Using `sha512` is certainly more thorough than sha1. I'm mainly wondering if we should converge on the `sha512` hashes all around to make sure unintentional lock file changes like this are avoided. (Sorry I always mention you for npm questions/conventions)

Reply:
No worries on the mention since I've spent so much time trying to understand the chaos of npm 😛 

As long as they're not changing back and forth, I think we can be fine with whatever. Based on [this](https://stackoverflow.com/questions/47638381/why-did-package-lock-json-change-the-integrity-hash-from-sha1-to-sha512), npm's been using sha512 for a few years now, and the web app's package-lock has all sha512 hashes as well. I'd assume those previous hashes were from an older version of npm